homeassistant/components/rachio/binary_sensor.py

```diff
@@ -94,7 +94,7 @@ def __init__(self, controller):
     @property
     def name(self) -> str:
         """Return the name of this sensor including the controller name."""
-        return f"{self._controller.name} online"
+        return f"{self._controller.name} status"
```

Comment:
```suggestion
        return self._controller.name
```

I think we can simply use the name since its `DEVICE_CLASS_CONNECTIVITY` its going to be `Connected` or `Not Connected`


```
cast/src/receiver/layout/hc-launch-screen.ts:          ${this.hass ? "Connected" : "Not Connected"}
device_trigger.py:CONF_CONNECTED = "connected"
device_trigger.py:CONF_NOT_CONNECTED = "not_connected"
```

Reply:
👍